browser.js

```diff
@@ -0,0 +1,176 @@
+'use strict';
+/* **
+ * @description A client side port of `@ebay/retriever`
```

Comment:
(minor) I think you can remove this comment block. This file is self-explanatory.